casbin/core_enforcer.py

```diff
@@ -294,7 +291,9 @@ def enforceEx(self, *rvals):
                 p_parameters = dict(zip(p_tokens, pvals))
                 parameters = dict(r_parameters, **p_parameters)
 
-                if util.has_eval(exp_string):
+                if not util.has_eval(exp_string):
+                    expression = self._get_expression(exp_string, functions)
+                else:
```

Comment:
`has_eval` will appear undefined error on line [331](https://github.com/casbin/pycasbin/pull/138/files#diff-ead3924a9f87932efcef6b5137980fc64185397aa14d188d3c1ccb58765dc287L331)

Reply:
This is my test code. It should not be include in this PR. I have removed this.